react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import { Header } from './Header';
+import { Image } from './Image';
+import { Caption } from './Caption';
+import { Description } from './Description';
+import { Button } from './Button';
+import './App.css';
+
+const srcStr = '/starry-sky.jpeg';
```

Comment:
Don't use a `str` suffix. It is redundant and makes the variable name harder to read.